azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/PermissionManager.kt

```diff
@@ -22,18 +23,29 @@ internal class PermissionManager(
     private lateinit var activity: Activity
     private lateinit var audioPermissionLauncher: ActivityResultLauncher<Array<String>>
     private lateinit var cameraPermissionLauncher: ActivityResultLauncher<String>
+    private lateinit var phonePermissionLauncher: ActivityResultLauncher<String>
     private var previousPermissionState: PermissionState? = null
 
-    private val audioPermission by lazy { getPermissionsList() }
+    private val audioPermissions =
+        arrayOf(
+            Manifest.permission.BLUETOOTH_CONNECT,
+            Manifest.permission.RECORD_AUDIO,
+            Manifest.permission.ACCESS_NETWORK_STATE,
+            Manifest.permission.WAKE_LOCK,
+            Manifest.permission.MODIFY_AUDIO_SETTINGS,
+            Manifest.permission.FOREGROUND_SERVICE
```

Comment:
`FOREGROUND_SERVICE` permission available only in API28 and newer
`BLUETOOTH_CONNECT` permissioin available only in API31 and newer